packages/data/undocumented-exports.md

```diff
@@ -0,0 +1,9 @@
+# Undocumented exports
```

Comment:
> https://github.com/WordPress/gutenberg/pull/14271#discussion_r263735768

In that case, this needs to be delete as well.

Reply:
:man_facepalming: afd9249